src/YarpUtilities/tests/YarpUtilitiesTestMain.cpp

```diff
@@ -0,0 +1,9 @@
+/**
+ * @file YarpUtilitiesMain.cpp
+ * @authors Giulio Romualdi
+ * @copyright 2019 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#define CATCH_CONFIG_MAIN
+#include <catch2/catch.hpp>
```

Comment:
Is this file supposed to be this empty?

Reply:
Yes. It is a trick to speed up the compilation phase when Catch2 is used ([Here](http://mochan.info/c++/2019/11/12/pre-compiled-headers-gcc-clang-cmake.html) further details)